File: android/apolloui/src/main/java/io/muun/apollo/presentation/ui/fragments/manual_fee/ManualFeeParentPresenter.java
package io.muun.apollo.presentation.ui.fragments.manual_fee;

import io.muun.apollo.domain.model.PaymentRequest;
import io.muun.apollo.presentation.ui.base.ParentPresenter;

public interface ManualFeeParentPresenter extends ParentPresenter {

    PaymentRequest getPaymentRequest();

    void confirmFee(double selectedFeeOption);
}

File: android/apolloui/src/main/java/io/muun/apollo/presentation/ui/fragments/manual_fee/ManualFeeParentPresenter.kt
package io.muun.apollo.presentation.ui.fragments.manual_fee

import io.muun.apollo.presentation.ui.base.ParentPresenter
import newop.EditFeeState
import rx.Observable

interface ManualFeeParentPresenter : ParentPresenter {

    fun watchEditFeeState(): Observable<EditFeeState>

    fun confirmFee(selectedFeeRateInVBytes: Double)
}